src/github.com/getlantern/flashlight/genconfig/cloud.yaml.tmpl

```diff
@@ -20,6 +20,9 @@ client:
     - domain: {{.Domain}}
       ipaddress: {{.IpAddress}}{{end}}
 proxiedsites:
+  delta:
+    additions: []
+    deletions: []
```

Comment:
This change just makes the generated yaml file actually work on the client side -- without this the client side won't load.